### PR TITLE
test addition/update for SDN fips

### PR DESCRIPTION
**Pull request by lenahorsley**
[PERFSCALE-2617](https://issues.redhat.com/browse/PERFSCALE-2617)

### CODE REVIEW

**Comment by lenahorsley** [comment]
@svetsa-rh 
@paigerube14 
@qiliRedHat

**Comment by lenahorsley** [comment]
/ pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-node-density-24nodes-sdn-fips

**Comment by lenahorsley** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-node-density-3nodes-sdn-fips

**Comment by lenahorsley** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-node-density-3nodes-sdn-fips

**Comment by lenahorsley** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-node-density-3nodes-sdn-fips

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml [hunk 1]
```diff
@@ -2577,6 +2577,88 @@ periodics:
...
+    org: openshift-qe
+    repo: ocp-qe-perfscale-ci
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-perfscale-qe
+    ci-operator.openshift.io/variant: aws-4.15-nightly-x86
+    ci.openshift.io/generator: prowgen
+    job-release: "4.15"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-node-density-24nodes-sdn-fips
```
Comment (paigerube14) [inline ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:2598]: Can you also add slack reporting after the name of this periodic?

```
reporter_config:
    slack:
      channel: '#ocp-qe-scale-ci-results'
      job_states_to_report:
      - success
      - failure
      - error
      report_template: '{{if eq .Status.State "success"}} :white_check_mark: Job *{{.Spec.Job}}*
        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> :white_check_mark:
        {{else}} :warning:  Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
        logs> :warning: {{end}}'
```

**Pull request merged**